content/account_management/faq/how-do-i-configure-azure-ad-as-a-saml-idp.md

```diff
@@ -13,64 +13,75 @@ further_reading:
 
-12. Enter in an email address at which you would like to be notified about maintenance issues
+    {{< img src="account_management/faq/notification_email.png" alt="Notification email" responsive="true" popup="true" style="width:70%;">}}
 
-13. Once you are back to the applications main page, navigate to "Attributes"
-    {{< img src="account_management/faq/Active_Directory_-_Microsoft_Azure6.png" alt="Active_Directory_-_Microsoft_Azure6" responsive="true" popup="true">}}
+13. Click at the bottom of the page on Step 5 **Configure DatadogSSO_test**.
 
-14. In "SAML Token Attributes", hover over the line where "TYPE" is "user attribute (nameid)" (usually the first one) and click on the pencil icon to edit
-    {{< img src="account_management/faq/Active_Directory_-_Microsoft_Azure7.png" alt="Active_Directory_-_Microsoft_Azure7" responsive="true" popup="true">}}
+14. Scroll down to the Step 3 of the *Configure DatadogSSO_test for single sign on* section, and download the **SAML XML Metadata** File.
 
-15. Change the attribute value to "user.mail" and press the check button
-    {{< img src="account_management/faq/Active_Directory_-_Microsoft_Azure14.png" alt="Active_Directory_-_Microsoft_Azure14" responsive="true" popup="true">}}
+15. Go at the top of your **SSO Configuration section** and click **Save**.
+
+16. Navigate back to [Datadog SAML page][1] and upload the **SAML XML Metadata** file downloaded in step 14:
+
+    {{< img src="account_management/faq/SAML_Configuration___Datadog10.png" alt="SAML_Configuration___Datadog10" responsive="true" popup="true" style="width:70%;">}}
+
+17. Make sure to press the **Upload File** button after having chosen the XML file to upload.
+
+18. And that’s it! It should now say *SAML is ready* and that Valid IdP metadata installed.
+
+    {{< img src="account_management/faq/SAML_Configuration___Datadog11.png" alt="SAML_Configuration___Datadog11" responsive="true" popup="true" style="width:70%;">}}
+
+19. Begin to log in to Datadog via Azure AD by pressing **Enable**:
+    
+    {{< img src="account_management/faq/SAML_Configuration___Datadog12.png" alt="SAML_Configuration___Datadog12" responsive="true" popup="true" style="width:70%;">}}
 
-16. Press "Apply Changes" at the bottom of the screen
-    {{< img src="account_management/faq/Active_Directory_-_Microsoft_Azure9.png" alt="Active_Directory_-_Microsoft_Azure9" responsive="true" popup="true">}}
 
-17. Navigate back to https://app.datadoghq.com/saml/saml_setup and upload the xml file downloaded in step 11:
-    {{< img src="account_management/faq/SAML_Configuration___Datadog10.png" alt="SAML_Configuration___Datadog10" responsive="true" popup="true">}}
+#### Optional 
 
-18. Make sure to press "Upload File" after choosing the XML file
+If you are using a SSO via a Datadog button or link, you need to add a Sign-on URL. To do this, navigate back to the configure SSO Configuration section of the Azure Application (Step 8) check off **Show advanced URL settings**:
```

Comment:
If you are using an SSO via a Datadog button or link, you need to add a Sign-on URL. To do this, navigate back to the SSO Configuration section of the Azure Application (Step 8) and check off **Show advanced URL settings**: